circle.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: 0.10.0
```

Comment:
What about testing on other versions of Node? Previously this was being run on Node `0.10`, `0.11`, `0.12`, `iojs`. Should probably remove `iojs` and add `4` and `5` to the tests to make sure this works with all versions of Node that it might be used on.